gremlin-python/src/main/python/gremlin_python/driver/driver_remote_connection.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, url, traversal_source="g", protocol_factory=None,
         # so that they can be closed if this parent session is closed.
         self.__spawned_sessions = []
 
-        if message_serializer is None:
+        if message_serializer is None and graphson_reader is not None and graphson_writer is not None:
```

Comment:
Did you encounter a case where graphson_read/graphson_writer are None?

Just wondering why this is required here but not in client.py

Reply:
this code is related to setting default serializer

Reply:
reverted in https://github.com/apache/tinkerpop/pull/1676/commits/ff4e16c18236af61491e680cbf810664aba38e56